implementation/src/main/java/com/teampc/controller/test/TestRowController.java

```diff
@@ -107,6 +104,7 @@ void onGradeActionHandler(ActionEvent event) {
           });
 
        } catch (IOException e) {
+          LOG.error("could not show grading for a test");
           e.printStackTrace();
```

Comment:
LOG.error can also take a throwable that will print a stack trace, this is generally preferable to e.printStackTrace().
```java
} catch (IOException e) {
   LOG.error("could not show grading for a test", e);
}
```